Makefile.work

```diff
@@ -247,6 +247,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            SONIC_IMAGE_VERSION=$(SONIC_IMAGE_VERSION) \
                            ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
                            ENABLE_ZTP=$(ENABLE_ZTP) \
+                           ENABLE_PDE=$(ENABLE_PDE) \
```

Comment:
why do we still have ENABLE_PDE? isn't that you have changed them to INCLUDE_PDE?

Reply:
Need to changed